package.json

```diff
@@ -4,7 +4,12 @@
   "version": "2.3.0",
   "homepage": "https://github.com/okta/okta-auth-js",
   "license": "Apache-2.0",
-  "main": "lib/index.js",
+  "main": "lib/serverIndex.js",
+  "browser": {
+    "./lib/serverIndex.js": "./lib/browserIndex.js",
```

Comment:
I'm afraid I do not know enough about this, but is the mapping of `serverIndex` to `browserIndex` correct?

Reply:
Yepp, it changes the "main" entry point from "serverIndex.js" to "browserIndex.js" (https://webpack.js.org/configuration/resolve/#resolve-aliasfields).

Reply:
@haishengwu-okta It would not, since `jquery/index.js` require `browser.js` https://github.com/okta/okta-auth-js/blob/55f59b3fad109bc837fb2e684dddc7e8efbddcf0/jquery/index.js#L15

Reply:
OT, but `okta-signin-widget` doesn't consume `okta-auth-js` 2.0 yet. So will get breaking changes but from previous updates.